timApp/document/translation/language.py

```diff
@@ -123,3 +123,13 @@ def to_json(self) -> dict:
             "flagUri": self.flag_uri,
             "name": self.autonym,
         }
+
+    # ignore type checking this for now, see: https://peps.python.org/pep-0484/#the-problem-of-forward-declarations
+    def __lt__(self, other) -> bool:  # type: ignore
```

Comment:
```suggestion
    def __lt__(self, other: "Language") -> bool:
```

MyPy ja PyCharm tunnistavat tyypin, jos se on merkitty merkkijonona. Python 3.11:ssa sitten on erikseen Self-tyyppi, mutta sitä odotellessa voidaan käyttää tuota merkkijonoversiota.